services/movie.py

```diff
@@ -0,0 +1,32 @@
+from db.models import Movie, Genre, Actor
+
+
+def get_movies(genres_ids: list[int], actors_ids: list[int]) -> Movie:
+    if not genres_ids or not actors_ids:
+        return Movie.objects.all()
+    if genres_ids or actors_ids:
+        return Movie.objects.filter(id__in=genres_ids).filter(id__in=actors_ids)
```

Comment:
The logic here is incorrect. The current condition will always be true if either `genres_ids` or `actors_ids` is provided, leading to incorrect filtering. Consider using a more precise condition to filter movies based on both genres and actors.